roachtests/awsdms: remove unrequired session setting

otan opened the pull request:
Now that schema changer removes the rowid column, we can no longer test
with the expect and ignore hidden columns setting.

Release note: None

Release justification: test only change

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/87392)
<!-- Reviewable:end -->

rafiss reviewed (approved):
nit: that's a session setting not a cluster setting, but lgtm since it's just a test

otan committed:
roachtests/awsdms: remove unrequired session setting

Now that schema changer removes the rowid column, we can no longer test
with the expect and ignore hidden columns setting.

Release note: None

Release justification: test only change

otan commented:
bors r=rafiss
